docs/modules/integrate/pages/legacy-file-connector.adoc

```diff
@@ -349,4 +349,24 @@ S3 sink uses the multi-part upload feature of S3 SDK. The sink buffers
 the items to parts and uploads them after buffer reaches to the
 threshold. The multi-part upload is completed when the job completes and
 makes the objects available on the S3. Since a streaming jobs never
-complete, S3 sink is not currently applicable to streaming jobs.
\ No newline at end of file
+complete, S3 sink is not currently applicable to streaming jobs.
+
+== Fvecs and ivecs
+[.enterprise]*{enterprise-product-name}*
+
+Fvecs and ivecs files are binary files containing, respectively, float and integer vectors with IDs. The connectors are similar to local file connectors, but work with fvecs and ivecs files.
+Fvecs and ivecs files can only be sources, and can therefore only be read; fvecs and ivecs files cannot be used as sinks.
+
+```java
```

Comment:
do we have plans for Python samples too?

Reply:
not here - Jet is only available in Java. 
AFAIK we did not implement fvecs/ivecs support in python, but there are some libraries that do that.